### PR TITLE
Utilize nameof operator

### PR DESCRIPTION
**Pull request by henkmollema**
Minor PR, just saw this passing.

### CODE REVIEW

File: test/Microsoft.Framework.DependencyInjection.Tests/Utils/MultiServiceHelpers.cs [hunk 1]
```diff
@@ -25,6 +25,11 @@ public static IEnumerable GetMultiService(Type collectionType, Func<Type, IEnume
 
         private static IEnumerable Cast([NotNull] IEnumerable collection, Type castItemsTo)
         {
+            if (collection == null)
```
Comment (hishamco) [inline test/Microsoft.Framework.DependencyInjection.Tests/Utils/MultiServiceHelpers.cs]: @henkmollema I don't we need the condition in this case, while ```collection``` parameter was already decorated with ```[NotNull]``` attribute
/cc @Eilon

Comment (henkmollema) [inline test/Microsoft.Framework.DependencyInjection.Tests/Utils/MultiServiceHelpers.cs]: I see, this was removed by your PR: 277d338643f439992892275126b49bf1d6e2579f
but I made this PR before yours was merged.

**Comment by hishamco** [comment]
My PR was #230 while your is #240?!!

**Comment by henkmollema** [comment]
Yes, but it was merged 4 days ago while I created my PR 6 days ago.

**Comment by hishamco** [comment]
Thanks for your contribution :)

**Pull request closed without merging**